### DISCUSSION TITLE
Add option to enable SSL using docker ENV variable

### DISCUSSION THREAD

**Discussion by scropion86** [wg-easy/wg-easy#1050] [Ideas]
Please add an easy way to enable SSL for the server since the current implementations in documentation is little bit complicated
and if the user already got his certificate and key, he could pass it as docker ENV variable on starting the docker server
and user will be responsible for attaching and renew the certificates

for example, portainer allow the user to select the certificate from the web UI (the own server WEBUI)

![image](https://github.com/wg-easy/wg-easy/assets/5569470/7f4c8cad-6e65-493d-adb0-ba7819356fa4)

other similar servers allow to attach it in docker ENV variable

![image](https://github.com/wg-easy/wg-easy/assets/5569470/e14b1f8a-7883-4d34-8be3-e0c2d121a661)

**Comment by pheiduck** [comment]
Open a PR. ;)
